build.gradle

```diff
@@ -212,6 +212,8 @@ dependencies {
   implementation group: 'io.github.openfeign', name: 'feign-jackson', version: '10.8'
   // end region
 
+  compile group: 'uk.gov.hmcts.reform', name: 'service-auth-provider-client', version: '3.0.0'
```

Comment:
- it's `implementation` now
- it's `3.1.1` now
- why can't we keep hmcts dependencies nicely in one group?